### DISCUSSION TITLE
Implementing a pyroll-basic GUI

### DISCUSSION THREAD

**Discussion by RollingTechSupport** [pyroll-project/pyroll-core#98] [Ideas]
Hi everybody, 
So as a lot of people criticize that we don't have a proper GUI @axtimhaus and I at least want to start discussing about what a good gui should look like to maybe find and fund someone who can do it. Lastly I discoverd the python-eel project which shows great potential in creating a local web based GUI. It is a direct link to CSS and HTML via Python.
So feel free to write down some ideas and If you maybe are familiar with HTML or CSS already provide some templates if you can and want.

**Comment by GRPlan** [comment]
I think that for the foreseeable future a well designed jupyter-notebook will be sufficient.

An example for a modern GUI (in terms of work required and functionality needed) i would suggest to have a look at PrePoMax https://prepomax.fs.um.si/

Another approach could be a workbench in FreeCad https://www.freecad.org/

**Comment by sag-kalibreur** [comment]
I now also think that a well-designed jupyter-notebook is sufficient, we have seen many suggestions recently to improve comfort and usability, this should be standardised a little more.

Nevertheless, I would not lose the focus on a GUI, you will reach many more users with it.
